insta-clone/src/components/PostContainer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+
+class PostContainer extends React.Component {
```

Comment:
I like the idea of using classes here. Thanks for the PR. When you get to it, add a tasklist to your description box. Good job for working on the MVP! 👍